.github/workflows/backport.yml

```diff
@@ -9,10 +9,6 @@ on:
   pull_request_target:
     types: [closed, labeled]
 
-concurrency:
-  group: backport-${{ github.workflow }}-${{ github.event_name }}-${{ github.event.pull_request.number || github.run_id }}
-  cancel-in-progress: true
```

Comment:
Shouldn't we keep the concurrency group (but without cancelling) so that the jobs run sequentially?

Reply:
> Job 3 comes in and replaces Job 2 in the queue

Huh?

Firstly, are we talking about **job** queues or workflow **run** queues? IIUC, concurrency groups can be used at either the job level or the workflow level to control jobs or workflow runs respectively.

Secondly, where have you observed that pushing an item onto the queue removes/replaces other queued items that have not started yet? That is __not__ the behaviour I would _expect_ in a concurrency queue.

(Thirdly, we might be using skipped and cancelled interchangeably, but they are distinct things as far as GHA is concerned)